pkg/ignite/create.go

```diff
@@ -49,12 +51,25 @@ func Create(name string, spec *config.Machine, pubKeyPath string) (id string, er
 func setupCopyFiles(copyFiles map[string]string) []string {
 	ret := []string{}
 	for k, v := range copyFiles {
-		s := fmt.Sprintf("--copy-files=%s:%s", k, v)
+		s := fmt.Sprintf("--copy-files=%s:%s", toAbs(k), v)
 		ret = append(ret, s)
 	}
 	return ret
 }
 
+func toAbs(p string) string {
+	ap := p
+	if !path.IsAbs(ap) {
```

Comment:
only use filepath? I think that library is the most cross-platform one.

also maybe invert the logic so that if the path is already absolute, return p directly, and if not return the absolute path relative to the directory that `FOOTLOOSE_CONFIG` is present in, or pwd